Lets Do This/Classes/LDTAppDelegate.m

```diff
@@ -30,7 +30,12 @@ @implementation LDTAppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     NSDictionary *keysDict = [NSDictionary dictionaryWithContentsOfFile:[[NSBundle mainBundle] pathForResource:@"keys" ofType:@"plist"]];
     NSDictionary *environmentDict = [NSDictionary dictionaryWithContentsOfFile:[[NSBundle mainBundle] pathForResource:@"environment" ofType:@"plist"]];
-    
+
+    [NRLogger setLogLevels:NRLogLevelNone];
+    if (environmentDict[@"NewRelicLoggerEnabled"] && [environmentDict[@"NewRelicLoggerEnabled"] boolValue]) {
+        [NRLogger setLogLevels:NRLogLevelError|NRLogLevelWarning];
```

Comment:
From NRLogger.h:
> /*******************************************************************************
 * Log levels used in the agent's internal logger
 *
 * When calling NRLogger setLogLevels: pass in a bitmask of the levels you want
 * enabled, ORed together e.g.
 *   [NRLogger setLogLevels:NRLogLevelError|NRLogLevelWarning|NRLogLevelInfo];
 *
 * NRLogLevelALL is a convenience definition.
 *
 * NRLogger's default log level is NRLogLevelError|NRLogLevelWarning
 *******************************************************************************/